generators/readme/bin/generate-readme.ts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+import mri from 'mri';
+import report from 'vfile-reporter';
+import { generate } from '../src/index';
+import { CommandFlags } from '../src/types';
+
+const flags = mri(process.argv.slice(2), {
+  alias: { help: ['h'], boolean: ['dry-run'] },
+});
+const [pathToPackage] = flags._;
```

Comment:
`pathToPackage` isn't defined!

Reply:
What do you mean? This is array destructuring... 🤔

Reply:
Looking at the docs of `mri` it seems that `_` field will have the args mentioned after `--` in cli command, which does not match the usage example given on line 17.
https://www.npmjs.com/package/mri#usage

Reply:
The docs are a bit misleading. The `_` contains all arguments that are NOT CLI options. For example

```
my-script foo bar baz --hello
```

The `_` field contains `['foo', 'bar', 'baz']`.